docs/content/users/quickstart.md

```diff
@@ -397,6 +397,33 @@ The Laravel project type can be used for [Lumen](https://lumen.laravel.com/) lik
     ddev config --project-type=laravel --docroot=public --omit-containers=db --disable-settings-management=true
     ```
 
+!!!tip "Add Vite support?"
+    Since Laravel v9.19.0, Vite will be shipped as default [asset bundler](https://laravel.com/docs/11.x/vite). There are two tweaks needed for usage in DDEV.
```

Comment:
```suggestion
    Since Laravel v9.19.0, Vite is included as the default [asset bundler](https://laravel.com/docs/11.x/vite). There are two tweaks needed for usage in DDEV.
```